Sources/Core/Transaction/CodableTransaction.swift

```diff
@@ -293,17 +294,14 @@ extension CodableTransaction: Codable {
 
 }
 
-public protocol Policyable {
-    func resolve(provider: Web3Provider, transaction: CodableTransaction?) async throws -> BigUInt
-}
-
 extension CodableTransaction {
     public enum GasLimitPolicy {
         case automatic
         case manual(BigUInt)
         case limited(BigUInt)
         case withMargin(Double)
```

Comment:
If we remove the old resolve functions right away, I'd move these policy enum's to their own file.

Reply:
Agreed with that.